samples/DotNet/Microsoft.Azure.ServiceBus/AutoForward/AutoForward.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net462;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>net461;netcoreapp3.0</TargetFrameworks>
```

Comment:
.NET Core 3.0 is [still in preview](https://dotnet.microsoft.com/download/dotnet-core).

Reply:
That's true. It is also a great test for .NET Core 3.0 Preview 4.

Reply:
People use samples to learn and apply in their environments for production purposes. I would suggest to keep the .NET Core with a stable version. Once v3 is graduates from preview, a separate PR for that would be good.

Reply:
Totally agree. How about .NET 4.6.2? I notice that is not supported.

Reply:
To be compliant with `netstandar2.0`, it would be better to use [4.7.2](https://dotnet.microsoft.com/download/dotnet-framework/net472). Want to do that and revert 3.0?

Reply:
Sure. It should be net472;netcoreapp2.2.

Reply:
@SeanFeldman The dependencies description for .NET4.6.1 in the nuget package should be updated also to .NET 4.7.2.

Reply:
Nuget package is listing the minimal .NET Framework requirement. Has nothing to do with the samples code.

Reply:
Yes, I confused the minimum with the maximum. Apologies for that.